Modules/mmapmodule.c

```diff
@@ -1463,9 +1463,9 @@ PyInit_mmap(void)
 #endif
 
     setint(dict, "PAGESIZE", (long)my_getpagesize());
-
+#ifndef __VXWORKS__
```

Comment:
I suggest to define my_getallocationgranularity to my_getpagesize above like on Unix.

Reply:
Your correct this is not required.